eta/core/serial.py

```diff
@@ -181,6 +190,43 @@ def write_pickle(obj, path):
         pickle.dump(obj, f)
 
 
+def serialize_numpy_array(array):
+    '''Serializes a numpy array.
+
+    Args:
+        array: a numpy array
+
+    Returns:
+        the serialized string
+    '''
+    #return pickle.dumps(array)
```

Comment:
There are 4 comments like this between `serialize_numpy_array` and `deserialize_numpy_array`. Can we remove these?

Reply:
I converted these to a more proper comment. I was leaving bread crumbs for other serialization strategies that I considered